homeassistant/components/modbus/__init__.py

```diff
@@ -268,21 +267,21 @@ def number(value: Any) -> Union[int, float]:
 
 SERVICE_WRITE_REGISTER_SCHEMA = vol.Schema(
     {
-        vol.Optional(ATTR_HUB, default=DEFAULT_HUB): cv.string,
-        vol.Required(ATTR_UNIT): cv.positive_int,
-        vol.Required(ATTR_ADDRESS): cv.positive_int,
-        vol.Required(ATTR_VALUE): vol.Any(
+        vol.Optional(CONF_HUB, default=DEFAULT_HUB): cv.string,
```

Comment:
Service constants should use `ATTR_*`.

Reply:
this means that a couple of constants will be double (like CONF_ADDRESS), but it is a more clean solution.

updated.